src/components/Card/indexCard.js

```diff
@@ -13,93 +13,85 @@ const IndexCalendar = dynamic(() => import('../Calendar/indexCalendar'), {
 import StyledHideButton from '../Buttons/HideButton/styled';
 import StyledCardBody from '../CardBody/styled';
 import StyledCardDescription from '../CardDescription/styled';
-import CardFooter from '../CardFooter';
+import StyledCardFooter from '../CardFooter/styled';
 import StyledCardHeader from '../CardHeader/styled';
 import StyledCardHeadline from '../CardHeadline/styled';
 
 import StyledCard from './styled';
 
-export default function IndexCard({card}) {
-	const changeSets = useStore(state => state.changeSets);
-	const changeReps = useStore(state => state.changeReps);
-	const changeWeight = useStore(state => state.changeWeight);
-	const bookmarked = useStore(state => state.bookmarked);
-	const [isShown, setIsShown] = useState(false);
-	const [showText, setShowText] = useState(false);
-	let [buttonText, setButtonText] = useState(true);
-	const [isBookmarked, setIsBookmarked] = useState(false);
-	const cards = useStore(state => state.cards);
-
-	const handleChange = () => {
-		return setButtonText(!buttonText);
-	};
-
-	function handleSubmit(event) {
-		event.preventDefault();
-	}
-	function handleSetInput(input) {
-		changeSets(card.id, Number(input));
-	}
-
-	function handleRepInput(input) {
-		changeReps(card.id, Number(input));
-	}
-
-	function handleWeightInput(input) {
-		changeWeight(card.id, Number(input));
-	}
+export default function IndexCard({exerciseCard}) {
+	const handleBookmark = useStore(state => state.handleBookmark);
+	const [isCalendarVisible, setIsCalendarVisible] = useState(false);
+	const [isInstructionVisible, setIsInstructionVisible] = useState(false);
+	const addToPlanner = useStore(state => state.addToPlanner);
+	const workouts = useStore(state => state.workouts);
+	const currentExercises = workouts.filter(workout => exerciseCard.name === workout.name);
+	const sortedCurrentExercises = currentExercises.sort(
+		(a, b) => new Date(b.date) - new Date(a.date)
+	);
+	const latestStats = sortedCurrentExercises.length
+		? sortedCurrentExercises[0]
+		: {sets: '0', reps: '0', weight: '0'};
```

Comment:
As far as I understood you are not using the `lateststats` anymore, so please remove these lines.

```suggestion
```